test/e2e/tests/confirmations/signatures/siwe.spec.ts

```diff
@@ -31,7 +31,6 @@ import {
 } from './signature-helpers';
 
 describe('Confirmation Signature - SIWE @no-mmi', function (this: Suite) {
-  this.timeout(200000); // This test is very long, so we need an unusually high timeout
```

Comment:
This was fix by @seaona to fix develop branch CI https://github.com/MetaMask/metamask-extension/pull/28801